scripts/_common.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+pkg_dependencies="mumble-server mailutils websockify"
```

Comment:
Is it worth to include websockify as dependency without mumble-web? Would it be easy to exclude it otherwise?